src/config.ts

```diff
@@ -197,3 +197,5 @@ export const getTokensForScope = (scope: SearchScope | undefined): NativeTokenIn
 
 export const getFiatCurrencyForScope = (scope: SearchScope | undefined) =>
   (scope ? paraTimesConfig[scope.layer]?.[scope.network]?.fiatCurrency : undefined) ?? 'usd'
+
+export const noFiatValues = process.env.REACT_APP_NO_FIAT_VALUES === 'true'
```

Comment:
I think it was already discussed in a prev PR. REACT_APP_SHOW_FIAT_VALUES and than don't need to use logical not in all places.